src/olympia/devhub/tests/test_forms.py

```diff
@@ -762,3 +763,11 @@ def test_error_invalid_user(self):
         self.form = EditThemeOwnerForm({'owner': 'omg@org.yes'},
                                        instance=self.instance)
         assert not self.form.is_valid()
+
+
+class TestDistributionChoiceForm(TestCase):
+
+    def test_lazy_choice_labels(self):
+        """Tests that the labels in `choices` are still lazy"""
+        fields = forms.DistributionChoiceForm.base_fields
+        assert isinstance(fields['choices'].choices[0][1], Promise)
```

Comment:
This isn't the greatest test because it doesn't reproduce the problem. It would be better to try a few locales and make sure the label changes. We would do this in ancient times by using [self.activate()](https://github.com/mozilla/addons-server/blob/master/src/olympia/amo/tests/__init__.py#L441), like this:
````python
def test_lazy_choice_labels(self):
    with self.activate('en-US'):
        # check en label
    with self.activate('de'):
        # check German label
````